src/Compilers/VisualBasic/Portable/Scanner/CharacterInfo.vb

```diff
@@ -488,4 +488,37 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Function
     End Class
 
+    Friend Module CharExts
+
+        <Runtime.CompilerServices.Extension>
+        Friend Function IsFrom(c As Char, c0 As Char, c1 As Char) As Boolean
+            Return (c = c0) OrElse (c = c1)
```

Comment:
@VSadov
The use of the `OrElse` adds only 2 IL bytes per a usage. So don't think that cost isn't too much.